pkg/docker/config/config.go

```diff
@@ -258,6 +329,24 @@ func RemoveAuthentication(sys *types.SystemContext, registry string) error {
 
 // RemoveAllAuthentication deletes all the credentials stored in auth.json and kernel keyring
 func RemoveAllAuthentication(sys *types.SystemContext) error {
+	helpers, err := sysregistriesv2.AllConfiguredCredentialHelpers(sys)
```

Comment:
Same here.  We need to collect errors and try to remove all credentials from all possible sources.

We should not return early in error cases.